### PR TITLE
fnm: Add version 1.22.8

### PR DESCRIPTION
**Pull request by Armaldio**

### CODE REVIEW

File: bucket/fnm.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.22.8",
+    "description": "🚀 Fast and simple Node.js version manager, built in Rust",
```
Comment (Ash258) [inline bucket/fnm.json:3]: - Who ever think it is good idea to use emojis in application description??
- Source code language is completely irrelevant

File: bucket/fnm.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "homepage": "https://github.com/Schniz/fnm",
+    "license": "GPL-3.0",
```
Comment (Ash258) [inline bucket/fnm.json:5]: Invalid spdx identifier

File: bucket/fnm.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Schniz/fnm/releases/download/v1.22.8/fnm-windows.zip",
+            "hash": "0a1a2d5008971722edac7dd84b00507ec668eb5a866cc4841ba2d550c759c4c3"
+        }
+    },
+    "notes": "Add \"fnm env --use-on-cd | Out-String | Invoke-Expression\" to your powershell profile to enable automatic version switch",
```
Comment (Ash258) [inline bucket/fnm.json:12]: Move notes after license property

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/fnm.json (hunk 1 above)
Comment (Armaldio) [reply bucket/fnm.json]: I've just copied the description straight from the original repo.
I've now editor the description

File: bucket/fnm.json (hunk 2 above)
Comment (Armaldio) [reply bucket/fnm.json]: Is it valid now ?

File: bucket/fnm.json (hunk 3 above)
Comment (Armaldio) [reply bucket/fnm.json]: Done

File: bucket/fnm.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.22.8",
+    "description": "Cross-platform Node.js version switcher",
+    "homepage": "https://github.com/Schniz/fnm",
+    "license": "GPLv3",
```
Comment (Ash258) [inline bucket/fnm.json:5]: Invalid SPDX identifier

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/fnm.json (hunk 4 above)
Comment (Armaldio) [reply bucket/fnm.json:5]: What is the good identifier then ?
`GPL-3.0-only` ?

**Comment by Ash258** [comment]
Move this PR into [main bucket](https://github.com/ScoopInstaller/Main/)

**Pull request closed without merging**